Calendar/BookingHelpers/BookingGenericFunctions.ps1

```diff
@@ -0,0 +1,99 @@
+﻿function SplitDomainFromEmail {
+    param([string] $email)
+    return [string]$email.Split("@")[1]
+}
+
+function SplitIdentityFromEmail {
+    param([string] $email)
+    return [string]$email.Split("@")[0]
+}
+
+function IsConsumerMailbox {
+    param([string]$identity)
+
+    try {
+        $consumerMb = Get-ConsumerMailbox  $identity -ErrorAction Ignore
+        return [boolean]$consumerMb.IsProsumerConsumerMailbox -or $consumerMb.IsMigratedConsumerMailbox -or $consumerMb.IsPremiumConsumerMailbox
+    } catch {
+        return $false #consumer mailbox throws error if domain mailbox
+    }
+}
+
+function IsGetMailboxAvailable {
+    if (Get-Command -Name Get-Mailbox -ErrorAction SilentlyContinue) {
+        Write-Host "Validated connection to Exchange Online..." -ForegroundColor Green
+    } else {
+        Write-Error "Get-Mailbox cmdlet not found. Please validate that you are running this script from an Exchange Management Shell and try again."
+        Write-Host "Look at Import-Module ExchangeOnlineManagement and Connect-ExchangeOnline."
+        exit
+    }
+}
+
+function WriteTestTitle {
+    param([string]$title, [System.Boolean]$Success, [string]$errorMessage, [bool]$writeMessageAlways = $false)
```

Comment:
Expand this out, like function below.

Reply:
We shouldn't have multiple parameter names in a single line. Please expand this out.